src/main/java/mSearch/tool/FileSize.java

```diff
@@ -1,106 +1,54 @@
 package mSearch.tool;
 
-import java.net.HttpURLConnection;
-import java.net.URL;
-import mSearch.Config;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
 
-public class FileSize {
-
-    final static int TIMEOUT = 3000; // ms //ToDo evtl. wieder kürzen!!
+import java.io.IOException;
 
+public class FileSize {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)